MVC/models/model.categories.js

```diff
@@ -1,9 +1,13 @@
 const db = require("../.././db/connection")
+const apiEndpoints = require("../../endpoints.json")
 
 exports.selectCategories = () => {
     return db
         .query(`SELECT * FROM categories;`)
         .then((result) => {
             return result.rows;
     })
+}
+exports.fetchApi = () => {
```

Comment:
In this case we do not need a model for this end point, we can do all of this within the controller :)